google/cloud/spanner/internal/time.h

```diff
@@ -42,26 +42,24 @@ std::chrono::nanoseconds FromProto(google::protobuf::Duration const& proto);
 /**
  * Convert a system_clock::time_point to a google::protobuf::Timestamp.
```

Comment:
nit: the comment does not match the function.

Reply:
Fixed.